web-app/src/pages/TeamDetails/TeamDetails.js

```diff
@@ -0,0 +1,215 @@
+    background: 'linear-gradient(180deg, #7BEA4E 0%, #5DBA37 100%)',
+    borderColor: '#57B531',
+  },
+  outdated: {
+    label: 'OUTDATED',
+    color: '#E3E3E3',
+    background: 'linear-gradient(180deg, #E3E3E3 0%, #C3C3C3 100%)',
+    borderColor: '#ABABAB',
+  },
+};
+
+class TeamDetails extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      page: 1,
+      pageSize: 15,
+      loaded: false,
+    };
+
+    this.handleLoadMore = _.debounce(this.handleLoadMore.bind(this), 300, {
+      trailing: false,
+      leading: true,
+    });
+  }
+
+  componentWillMount() {
+    const { fetchTeam, match: { params } } = this.props;
+    fetchTeam(params.id);
+  }
+
+  handleTrashpointClick = id => {
+    if (!id) {
+      return;
+    }
+    this.props.history.push(`/trashpoints/${id}?focus=y`, { selectedArea: this.props.selectedArea });
+  };
+
+  handleLoadMore = () => {
+    if (!this.props.canLoadMore) {
+      return;
+    }
+    const { page, pageSize } = this.state;
+    this.setState(
+      {
+        page: page + 1,
+      },
+      () => {
+        this.props.fetchTrashpoints({
+          areaId: this.props.selectedArea.id,
+          pageNumber: this.state.page,
+          pageSize,
+        });
+      },
+    );
+  };
+
+  renderItems() {
+    const trashpoints = this.props.team.lastTrashpoints || [];
+    return trashpoints.map(t =>
+      (<TrashpointListItem
+        title={t.name}
+        {...t}
+        key={t.id}
+        handleClick={this.handleTrashpointClick}
+      />),
+    );
+  }
+
+  renderStatusCounts = () => {
+    const statusCounts = this.props.team.groupCount;
+    if (!statusCounts) {
+      return null;
+    }
+
+    return (
+      <div className="StatusCounts-container">
+        {_.map(STATUS_COUNT_HASH, (status, key) => {
+          const count = statusCounts[key] || 0;
+          return (
+            <div className="StatusCount-container" key={key}>
+              {console.log({ status, count })}
+              <div
+                className="StatusCount-circle"
+                style={{
+                  border: `1px solid ${status.borderColor}`,
+                  background: status.background,
+                }}
+              >
+                <span className="StatusCount-circle-label">
+                  {count}
+                </span>
+              </div>
+              <span
+                style={{ color: status.color }}
+                className="StatusCount-label"
+              >
+                {status.label}
+              </span>
+            </div>
+          );
+        })}
+      </div>
+    );
+  };
+
+
+  handleBackClick = () => {
+    this.props.history.push('/teams');
+  };
+
+  oneOrMany = (number, str) => (number === 0 || number) && (number === 1 ? `${number} ${str}` : `${number} ${str}s`);
+
+  render() {
+
+    const { team, match: { params } } = this.props;
+
+    return team.id === params.id ?
+      <div className="AreaList">
```

Comment:
different quotes for classnames. let's use single quotes or double quotes for every tag